src/omv/imlib/imlib.c

```diff
@@ -423,28 +423,92 @@ void imlib_fill_image_from_float(image_t *img, int w, int h, float *data, float
     }
 }
 
+// Packs src into dst. src must be an array of dst->w*dst->h*dtype*channels bytes, where channels is
+// 1 for grayscale and 3 for RGB. Note that scale/mean/stdev are only applied to the input image if dtype is 'f'.
+void imlib_pack(image_t *dst, void *src, const char dtype, float *scale, float *mean, float *stdev) {
+    // src will be unpacked into dst in forward order so that we can handle in-place packing.
+    int size = dst->w * dst->h;
+
+    int shift = (dtype == 'B') ? 0x00 : 0x80;
+    float fscale = 255.0f / (scale[1] - scale[0]);
+    float fadd = -scale[0] * fscale;
+    float fscale_r = fscale, fadd_r = fadd;
+    float fscale_g = fscale, fadd_g = fadd;
+    float fscale_b = fscale, fadd_b = fadd;
+
+    // To unnormalize the input image we need to apply the inverse of what pack does.
+    // We can do this by applying the unnormalization to fscale and fadd outside the loop.
```

Comment:
```suggestion
// The input image needs to be unnormalized by applying the inverse of what pack does.
// This can be achieved by applying the unnormalization to fscale and fadd outside the loop.
```
Please fix the comments tone when possible.

Reply:
pack() was eliminated.